Documentation/index.rst

```diff
@@ -88,9 +90,10 @@ get started and experiment with Cilium.
    :maxdepth: 2
    :caption: Security
 
+   security/index
    security/network/index
    security/policy/index
-   security/tutorial-toc
```

Comment:
This is in the 2nd commit, but should be renamed (possibly moved up at the same time) when the page is renamed, in the 1st commit.